data_validation/__main__.py

```diff
@@ -234,9 +236,10 @@ def build_config_from_args(args: Namespace, config_manager: ConfigManager):
             )
         config_manager.append_allow_list(args.allow_list, args.allow_list_file)
 
-    # Append CUSTOM_QUERY configs
+    # Append configs specific to CUSTOM_QUERY (i.e. query strings or strings from files)
     if config_manager.validation_type == consts.CUSTOM_QUERY:
-        config_manager.append_custom_query_type(args.custom_query_type)
+        if config_manager.validation_type == consts.CUSTOM_QUERY:
+            config_manager.append_custom_query_type(args.custom_query_type)
```

Comment:
As of now this condition is right after an exact same one, this can be misleading and maybe be seen as an error/duplicate line. At least, it was my first thought before I read the entire function and really understood its actual purpose. 

I think it's better to place it at the end of the function.

Reply:
Fixed